site/src/examples/progress/useProgressingValue.tsx

```diff
@@ -0,0 +1,53 @@
+import { useCallback, useEffect, useState } from "react";
+
+export function useProgressingValue(updateInterval = 100) {
```

Comment:
Put this in the same file as the example using it, so can be read together on the doc